nodeup/pkg/model/kube_apiserver.go

```diff
@@ -678,6 +678,7 @@ func (b *KubeAPIServerBuilder) buildPod(ctx context.Context, kubeAPIServer *kops
 			"/usr/local/bin/kube-apiserver",
 		}
 		container.Args = append(container.Args, sortedStrings(flags)...)
+		container.Args = sortServiceAccountIssuers(container.Args, kubeAPIServer.ServiceAccountIssuers)
```

Comment:
needed because otherwise the issuers will be sorted just by sortedStrings. We want the order to be correct, like it was specified in the configuration

Reply:
We also have lots of places where we use `Additonal...` to add secondary fields. This seems like one of those places, because the order matters.